### PR TITLE
Oy2 9915 - APPCH

### PR DESCRIPTION
**Pull request by daniel-belcher**
Adds APP-CH measure

Includes changes to OMS totals from WCC-CH 

Please refer to [this comment](https://qmacbis.atlassian.net/browse/OY2-17382?focusedCommentId=108587) in regards to OMS calculate totals button

[ticket](https://qmacbis.atlassian.net/browse/OY2-9915?atlOrigin=eyJpIjoiNzQ4ZmIzNjM4OTViNDIyYjg0OTU1N2YyNjkzY2UxNzYiLCJwIjoiaiJ9)

[instance](https://de19dyj8n5k6f.cloudfront.net/)

#### Pull Request Creator Checklist

- [ ] This PR has an associated issue or issues.
- [ ] The associated issue(s) are linked above.
- [ ] This PR meets all acceptance criteria for those issues.
- [ ] This PR and linked issue(s) are adequately documented
- [ ] This PR and linked issues(s) are a complete description of the changeset; an individual or team should be able to understand the issue(s) and changes by reading through this PR and it's links, with no further interaction.
- [ ] Someone has been assigned this PR.
- [ ] At least one person has been marked as reviewer on this PR.

#### Pull Request Reviewer/Assignee Checklist

- [ ] This PR has an associated issue or issues.
- [ ] The associated issue(s) are linked above.
- [ ] This PR meets all acceptance criteria for those issues.
- [ ] This PR and linked issue(s) are adequately documented
- [ ] This PR and linked issues(s) are a complete description of the changeset; an individual or team should be able to understand the issue(s) and changes by reading through this PR and it's links, with no further interaction.

### CODE REVIEW

File: services/ui-src/src/measures/2021/APPCH/index.tsx [hunk 1]
```diff
@@ -0,0 +1,68 @@
+import { useEffect } from "react";
+import { useFormContext } from "react-hook-form";
+import * as CMQ from "measures/CommonQuestions";
+import * as PMD from "./data";
+import { validationFunctions } from "./validation";
+import { getPerfMeasureRateArray } from "measures/globalValidations";
+import * as QMR from "components";
+import { FormData } from "./types";
```
Comment (jphechter) [inline services/ui-src/src/measures/2021/APPCH/index.tsx:8]:
```suggestion
import * as CMQ from "measures/CommonQuestions";
import * as PMD from "./data";
import * as QMR from "components";
import { FormData } from "./types";
import { getPerfMeasureRateArray } from "measures/globalValidations";
import { useEffect } from "react";
import { useFormContext } from "react-hook-form";
import { validationFunctions } from "./validation";
```
Alphabetized

**Review by jphechter — APPROVED** [review]
Approving with the caveat that I didn't take a super close look at the WCC-CH files, including the NDR stuff. Assuming that's here as a jumping off point and will be caught up with anything in that branch.

**Review by benjaminpaige — APPROVED** [review]
🔥🔥🔥🔥

**Comment by abelzer212** [comment]
Looks like you're including WCC-CH in this PR too, was that intentional?

**Comment by abelzer212** [comment]
Screen Shot 2022-04-05 at 8 20 17 AM
Screen Shot 2022-04-05 at 8 20 11 AM
Missing a validation that checks that the total's rate field is equal to what it has been calculated as. Here it would need to be 100
This also needs to be added to OMS. I think JP has added this in one of his measures if you want to look for it in the global validations or just ask him

**Comment by abelzer212** [comment]
There are several merge conflicts that also need to be resolved before this moves to quality. I would leave a note in the ticket that AMR-AD, and AMR-CH, and PPC-AD all need to be re-tested for the new total functionality.

**Pull request closed without merging**